requirements.txt

```diff
@@ -1,3 +1,7 @@
+sparse
+tensorflow
+torch
+mxnet
```

Comment:
I didn't include these on purpose: they are optional.
We want to maintain minimal dependencies.

Reply:
yes, that's right.